tests/tests.rs

```diff
@@ -498,6 +498,92 @@ impl<'a> ClientTest<'_> {
         spawned_task.await;
         assert_eq!(receiver.recv().await.unwrap(), true);
     }
+
+    async fn copy_object(&self) {
+        let src_object_name = rand_object_name();
+
+        let size = 16_usize;
+        self.client
+            .put_object(
+                &mut PutObjectArgs::new(
+                    &self.test_bucket,
+                    &src_object_name,
+                    &mut RandReader::new(size),
+                    Some(size),
+                    None,
+                )
+                .unwrap(),
+            )
+            .await
+            .unwrap();
+
+        let object_name = rand_object_name();
+        self.client
+            .copy_object(
+                &CopyObjectArgs::new(
+                    &self.test_bucket,
+                    &object_name,
+                    CopySource::new(&self.test_bucket, &src_object_name).unwrap(),
+                )
+                .unwrap(),
+            )
+            .await
+            .unwrap();
+
```

Comment:
Better to verify the size of the object after copy..

Reply:
Done